### PR TITLE
Feat/staking announcement

### PR DESCRIPTION
**Pull request by jjBlockchain**
## Description (optional)
* add staking promo banner

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/StakingBanner/StakingBanner.template.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+import { Image, Text } from 'blockchain-info-components'
+
+import { CloseLink } from '../styles'
+import { CardWrapper } from './StakingBanner.model'
+
+const StakingBanner = ({ onClickClose, rate }) => (
+  <CardWrapper>
+    <Padding horizontal={1} vertical={1.5}>
+      <Flex flexDirection='row' alignItems='center' justifyContent='space-between' gap={16}>
```
Comment (mperdomo-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/StakingBanner/StakingBanner.template.tsx:22]: Is this flex container needed? I has the default flex direction, reads space between but has only one item, and I think the align items can be set as `align-self` if needed in the flex below

File: packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/StakingBanner/StakingBanner.template.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                id='home.banners.stakingBanner.title'
+                defaultMessage='Earn up to {rate}% by staking ETH'
+                values={{ rate }}
+              />
+            </Text>
+
+            <Text color='grey600' weight={500} size='12px'>
+              <FormattedMessage
+                id='home.banners.stakingBanner.body'
+                defaultMessage='Stake your ETH today to earn {rate}% and unlock 10x lower staking fees through 2022!'
```
[the remark this answers is not in the document]
Comment (jjBlockchain) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/StakingBanner/StakingBanner.template.tsx:37]: will bring up with product

File: packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/StakingBanner/StakingBanner.template.tsx (hunk 1 above)
Comment (jjBlockchain) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/StakingBanner/StakingBanner.template.tsx:22]: @mperdomo there are 2 items in this flex.

File: packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/StakingBanner/StakingBanner.template.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jjBlockchain) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/StakingBanner/StakingBanner.template.tsx:37]: so we do want to add this and just remove this announcement later.

**Pull request merged**